src/index.html

```diff
@@ -138,6 +138,15 @@ <h4 class="card-title">
                 >
                   Send
                 </button>
+
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="sendEIP1559Button"
+                  disabled
```

Comment:
If we switch to a network that supports EIP-1559 without connecting to an account new button is enabled state.

<img width="1188" alt="Screen Shot 2021-06-24 at 5 52 04 PM" src="https://user-images.githubusercontent.com/43930900/123337275-f6b43700-d514-11eb-8d47-7166ff0567e7.png">

Reply:
Responded in latest commit :D thanks @NiranjanaBinoy